### DISCUSSION TITLE
Scheduling Events to take place in a Space

### DISCUSSION THREAD

**Discussion by docweirdo** [sleede/fab-manager#250] [Q&A] [answered]
Hi Fab-manager team,
I am in search for a tool to organize booking of classes and the management of rooms in our small school and am very excited by your project. However I was wondering whether it is possible to create an event that takes place at one of the spaces configured and also takes up the whole contingent for the duration of the event.
Thanks for the awesome work!

**Comment by sylvainbx** [comment, marked as answer]
Hi docweirdo,
Thanks for your interest in Fab-manager. Currently, you cannot associate an event with a space, those are two differents things:
- spaces are resources that can have availability slots scheduled in the agenda. Then, members can book these slots up to the maximum number of available places.
- events are designed to be more like a ticketing tool

However, why do you want to associate these two things? What are you trying to achieve this way? Also, I can't get what do you mean by "the whole contingent"?

**Reply by docweirdo** [reply]
Hi! Thanks for the reply.
I'm asking because my use case would require the ability to host or hold an event in a space, therefore associated with a space/room. The event would take place at said space and would either block booking of available slots for the duration of the event or take up "the whole contingent" of available slots.
I guess that is a different type of requirement and a different type of software I'm looking for though. I've gone through my requirements again and I think the booking of individual slots in spaces is less frequent in my case than the booking of whole spaces.
Thank you for developing cool open source software!

FYI: Don't mark a comment as an answer while typing a reply in Github discussions, it will get lost :)
